src/components/Organisms/MarketingPreferencesDS/_MarketingPrefsConfig.js

```diff
@@ -65,6 +64,8 @@ const buildValidationSchema = overrideOptions => {
   // Override with any custom options supplied
   const mpValidationOptions = merge(defaultOptions, overrideOptions);
 
+  const phoneRegex = /^(((((\+44)|(0044))\s?\d{4}|\(?0\d{4}\)?)\s?\d{3}\s?\d{3})|((((\+44)|(0044))\s?\d{3}|\(?0\d{3}\)?)\s?\d{3}\s?\d{4})|((((\+44)|(0044))\s?\d{2}|\(?0\d{2}\)?)\s?\d{4}\s?\d{4}))(\s?\\#(\d{4}|\d{3}))?$/;
```

Comment:
This looks like it accepts only _UK_ phone numbers, is that right?

In which case, we ought to say in the validation messages `Please enter a valid UK phone number` rather than imply that international numbers are invalid.

Reply:
Yeah okay cool, although we'd know need to do the same over in the FSU 

@KrupaPammi - that okay for you?

Reply:
Thanks @KrupaPammi